pkg/migrate/migrate_test.go

```diff
@@ -2796,7 +2797,6 @@ func Test_scaleDownPods(t *testing.T) {
 			req.Equal(tt.wantPods, actualPods)
 			req.Equal(tt.wantDeployments, actualDeployments)
 			req.Equal(tt.wantSS, actualSS)
-			req.Equal(tt.wantMatchingPVCs, actualMatchingPVCs)
```

Comment:
it might be good to remove the test entries for wantMatchingPVCs too since they're not used elsewhere